docs/tutorials/cmip6-recipe.ipynb

```diff
@@ -62,25 +62,25 @@
      "output_type": "stream",
```

Comment:
<p>Same comment on the link syntax.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pangeo-forge/pangeo-forge-recipes/pull/125/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
Same comment as above re: my myst directive confusion